cmd/user-agent/package-lock.json

```diff
@@ -1046,9 +1046,9 @@
       }
     },
     "@trustbloc-cicd/aries-framework-go": {
-      "version": "0.1.4-snapshot-93ff37e",
-      "resolved": "https://npm.pkg.github.com/download/@trustbloc-cicd/aries-framework-go/0.1.4-snapshot-93ff37e/d39b2b9ccafcba8a40080b3bfc3689d9b7f1be10f2571627e9e6253d7ece58f4",
-      "integrity": "sha512-9fBvlzSuLBripNp5htIt8N7diphqES7RVHZvywrZjROwtwohmYrLisYSKKR357Qw6R5q1iTofNRcJuIHs2q5aQ==",
+      "version": "0.1.4-snapshot-0e1dd13",
+      "resolved": "https://npm.pkg.github.com/download/@trustbloc-cicd/aries-framework-go/0.1.4-snapshot-0e1dd13/5fbd2137fe684d212edaabf389bf1aa5a306ada71b978bc99ae295fdf5c53274",
+      "integrity": "sha512-DkvchvmoQGzlMo9VUWE+ygbc7W1t92a+f+jufNUtJMYv6Br4rFLjNk1Lbf9hBfv5758ut+m3kC8SiD02ZJw0Dg==",
       "requires": {
         "axios": "0.19.2",
         "minimist": "^1.2.5"
```

Comment:
Should I commit this? I noticed that this file seems to get updated a lot when I `make` various targets...

Reply:
I reverted it. Let me know if I should have done something else.